include/open62541/plugin/nodestore.h

```diff
@@ -20,6 +21,10 @@
 #include <open62541/server.h>
 #include "ziptree.h"
 
+#if defined UA_ENABLE_ENCODE_AND_DUMP || defined UA_ENABLE_USE_ENCODED_NODES
```

Comment:
The nodestore is a plugin. The server only uses the plugin API.
You can implement the binary nodestore in a plugin without changing this API.

Please, not a single line should change in nodestore.h.

Encapsulation/hiding of internals is important to keep the architecture clean.